src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -0,0 +1,266 @@
+import argparse
```

Comment:
CAn you add a test file that at least imports this and runs a noop test so it gets included in the coverage calcs? Should've thought of this earlier

Reply:
writing automated tests for this is going to be difficult...

Reply:
👍 we can maybe just do unit testing for all functions and ignore `main`.

Reply:
:+1: